### PR TITLE
Mingione-NowCasting

### PR DESCRIPTION
**Pull request by minmar94**
Thanks for contributing your submission to the e-Rum2020 CovidR contest.

**For CovidR contributors:**

- [x] The title of this PR reflects your contribution
- [x] The pull request only includes one new `.yml` file under `contest/contributions`
- [x] Make sure "Allow edits from maintainers" is checked
- [x] Send an email to covidrcontest@erum.io (for further direct communication down the road)
     - Subject: "Submission to CovidR: PR #00", where #00 is the number of this Pull Request
     - Content: A full link to this Pull Request
     - Optionally include your social media profiles (Twitter, LinkedIn, Facebook): we will use them to `@mention` you while sharing the contribution through our channels.

---

For CovidR maintainers:

- [x] Label the PR as "submission"
- [x] Check the validity of the submission
- [x] Assess completeness of metadata and possibly adjust the `.yml` file-name
- [x] Check the integration of the new page in the website gallery (via Actions' artifact or locally)
- [x] Make sure the email to covidrcontest@erum.io was received
- [x] Add a field with the current `date` and approx. merge time (e.g. `2020-04-01-16:30` for April 1st, 4:30 PM)
- [x] Squash-merge the pull request (commit headline will be OK, consider cleaning-up the details)
- [x] The initial comment should be automatically created, otherwise comment manually with the raw content of [vote-comment.md](../tree/master/contest/_tools/vote-comment.md) and an initial like
- [x] Provide here the link to the contribution as published to the gallery: https://milano-r.github.io/erum2020-covidr-contest/mingione-nowcasting.html
- [x] Instruct contributors on how to include their badge from the gallery page, share on social media, and get a [free ticket](https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686).
- [x] Consolidate e-mailed information.

### CODE REVIEW

**Comment by minmar94** [comment]
Please let me know if you need anything else by my side. I hope that everything is ok with the submission. Thank you!

File: contest/contributions/mingione-nowcasting.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Contribution's title, author, abstract
+title: Nowcasting Italian Covid-19 Epidemic
+author: Marco Mingione, Pierfrancesco Alaimo Di Loro # comma-separated
+abstract: |
+  Our work is intended to give people an easy tool for accessing information
+  about the Italian COVID-19 epidemic in an interactive and transparent way.
+  The web application we built is automatically updated every day after the Italian Protezione Civile report (Data Source)
```
Comment (riccardoporreca) [inline contest/contributions/mingione-nowcasting.yml]: @minmar94 , did you mean to provide a link to `(Data Source)`? You can use the Markdown syntax
`([Data Source](https://data-source-url))`
Other than this we will be happy to finalize the submission process.

```suggestion
  The web application we built is automatically updated every day after the Italian Protezione Civile report ([Data Source](https://github.com/pcm-dpc/COVID-19))
```

Comment (minmar94) [reply contest/contributions/mingione-nowcasting.yml]: Yes of course! Should I modify the .yml file directly?

Comment (minmar94) [reply contest/contributions/mingione-nowcasting.yml]: [Italian Protezione Civile](https://github.com/pcm-dpc/COVID-19)

Comment (minmar94) [reply contest/contributions/mingione-nowcasting.yml]: Is this ok for you?

Comment (riccardoporreca) [reply contest/contributions/mingione-nowcasting.yml]: @minmar94, I can take care of it, no worries. I am about to finalize the process soon.

File: contest/contributions/mingione-nowcasting.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ 
+# Short name to be used in the website Gallery menu (can be the same as title)
+menu_entry: NowCastingCOVID-19
+# URL of the public source code repository
+repository: https://github.com/minmar94/StatGroup19
+# Type of contribution and how it can be featured as gallery content
+# Keep only the type and content relevant to your contribution
+# For a Shiny app, provide its URL:
+type: shiny
+content: https://statgroup19.shinyapps.io/NowCastingCovid19/
```
Comment (riccardoporreca) [inline contest/contributions/mingione-nowcasting.yml:18]: Include acceptance `date`
```suggestion
content: https://statgroup19.shinyapps.io/NowCastingCovid19/
date: 2020-05-21-00:30
```

**Review by riccardoporreca — APPROVED** [review]

**Comment by riccardoporreca** [comment]
@minmar94, your submission is now accepted and visible at https://milano-r.github.io/erum2020-covidr-contest/mingione-nowcasting.html :tada: 
You can click on the **eRum2020::Covidr badge** in the page to obtain the badge code you can embed in your repository.

Furthermore, feel free to spread your contribution on social media with the link above, using hashtags `#erum2020` `#covidr`, and invite the community to like the contribution using the corresponding button.

Last but not least: You can get your **free ticket for the pre-conference event** at https://www.eventbrite.it/e/covidr-erum-2020-pre-conference-event-tickets-104133026686

**Pull request merged**